web/cobrands/bexley/base.scss

```diff
@@ -3,6 +3,7 @@
 @import "../sass/mixins";
 @import "../sass/base";
 @import "../sass/waste";
+@import "../fixmystreet-uk-councils/societyworks-footer";
```

Comment:
Was going to ask, does this not affect the standard Bexley site? But it looks like the live site does not have a footer, when it should...(?)

Reply:
Yeah it also adds the footer to FMS, as it was missing from there.